test/drivers/README.md

```diff
@@ -0,0 +1,93 @@
+# Drivers tests
+
+## Rationale
+
+This test suite should allow you to check the behavior of our 3 drivers: `modern_bpf`, `bpf`, `kernel module`. To assert against the output of our drivers we use the so-called `scap-engines`.
+You don't have to build all the engines if you want to assert only some of the drivers, for example, the minimal build command is:
+
+```bash
+cmake -DUSE_BUNDLED_DEPS=On -DENABLE_DRIVERS_TESTS=On -DBUILD_LIBSCAP_GVISOR=Off -DCREATE_TEST_TARGETS=On ..
+```
+
+In this case, only the `kmod` engine will be built and you can assert only the behavior of the kernel module. If you want to assert also the bpf probe you have to add `-DBUILD_BPF=True`, while if you want to add the modern bpf probe engine you have to use `-DBUILD_LIBSCAP_MODERN_BPF=On`.
```

Comment:
I dream a future where:
* no driver is built by default
* we use same naming for the cmake options to enable drivers, like `DBUILD_LIBSCAP_MODERN_BPF`, `DBUILD_LIBSCAP_BPF` and `DBUILD_LIBSCAP_KMOD`

Reply:
you are a dreamer my friend!